packages/angular-material/src/controls/toggle.renderer.ts

```diff
@@ -41,7 +41,6 @@ import {
         <mat-slide-toggle
             (change)="onChange($event)"
             [checked]="isChecked()"
-            [disabled]="!enabled"
```

Comment:
I think we have the same issue here as with the checkbox renderer, the toggle is not controlled by the reactive control